relnotes/makelayout.feature.md

```diff
@@ -0,0 +1,37 @@
+* `ocean.util.log.Config`
+
+    The `configureLogger` and `configureOldLoggers` functions
```

Comment:
The way this is described "have gained" makes this sound like a breaking change. It should be made clear that these are overloads, not replacements.